test/test262-parser-tests/parser-tests.ts

```diff
@@ -19,6 +19,10 @@ const skipped = {
     '79f882da06f88c9f.js',
     '92b6af54adef3624.js',
 
+    // class fields
+    '98204d734f8c72b3.js',
+    'ef81b93cf9bdb4ec.js',
```

Comment:
test262-parser-tests seems quite out of date. The two files contain perfect valid JS code with class property definition.

```js
// 98204d734f8c72b3.js
(class {a})
// ef81b93cf9bdb4ec.js
(class {a=0})
```